modules/cats-effect/rules/src/main/scala/org/typelevel/fix/UnusedIO.scala

```diff
@@ -82,8 +82,8 @@ class UnusedIO extends SemanticRule("TypelevelUnusedIO") {
           }.asPatch
         case Term.Block(stats) =>
           stats.collect {
-            case stat if self.isDefinedAt(stat) =>
-              checkInner(stat)
+            case st if self.isDefinedAt(st) =>
+              checkInner(st)
```

Comment:
Now tests fail with
```
###########> Lint       <###########
===========> Mismatch  <===========

Obtained: scala/fix/UnusedIOTests.scala:239:8: error: [TypelevelUnusedIO.unusedIO]:
This IO expression is not used.
    s"${IO.pure("foo")}" /* assert: TypelevelUnusedIO.unusedIO
       ^^^^^^^^^^^^^^^^
Expected: scala/fix/UnusedIOTests.scala:239:9: error:
        This IO expression is not used. 
    s"${IO.pure("foo")}" /* assert: TypelevelUnusedIO.unusedIO
        ^^^^^^^^^^^^^^
Diff:
    s"${IO.pure("foo")}" /* assert: TypelevelUnusedIO.unusedIO
        ^-- asserted
       ^-- reported
```
And this makes me bewildering, to be honest.

Reply:
I think you just need to extend the `^` out a bit further to the left and right to satisfy it.
https://github.com/typelevel/typelevel-scalafix/blob/07ae8aa1a25324e827801ceedd85e013a63ef5b2/modules/cats-effect/input/src/main/scala/fix/UnusedIOTests.scala#L239-L240

Reply:
Yeah, but I struggle to understand what's changed that turned this test failed 🤷🏻

Reply:
Anyway, CI is now green, so let's move on

Reply:
> Yeah, but I struggle to understand what's changed that turned this test failed 🤷🏻

I wonder if it's related to this change.
- https://github.com/scalameta/scalameta/issues/3594